src/motion_planning/scripts/task_state.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+
+import rospy
+
+class TaskState(object):
+    STATE_TOPIC = 'state'
+    DESIRED_POSE_TOPIC = 'controls/desired_pose_global'
+    DESIRED_TWIST_GLOBAL_TOPIC = 'controls/desired_twist_global'
+    DESIRED_TWIST_LOCAL_TOPIC = 'controls/desired_twist_local'
+
+    # Initialise static members that will hold the publishers and listeners
+    STATE_LISTENER = None
```

Comment:
While it is true that there will probably only be one TaskState at a time, these are not meant to be static variables

Reply:
Why not? They are static because this is the cleanest way to implement a singleton in python.